pylint/checkers/spelling.py

```diff
@@ -35,8 +35,11 @@
         WikiWordFilter,
         get_tokenizer,
     )
+
+    PYENCHANT_AVAILABLE = True
 except ImportError:
     enchant = None
```

Comment:
```suggestion
```
Does this line serve a purpose anymore now that the `PYENCHANT_AVAILABLE` is here?

Reply:
Yeah it's basically so it's defined and we don't get screamed at by linters 😅 The design of this is peculiar, I'm thinking an external plugin would make more sense, because then it could convey in pip metadata that it NEED py-enchant and we could remove all of this. Maybe something to do as a breaking change in 4.0.